src/mscorlib/corefx/SR.cs

```diff
@@ -595,4 +595,29 @@ internal static string ArgumentException_ValueTupleLastArgumentNotATuple
     {
         get { return Environment.GetResourceString("ArgumentException_ValueTupleLastArgumentNotATuple"); }
     }
+
+    internal static string SpinLock_TryEnter_ArgumentOutOfRange
```

Comment:
The inliner should be smart enough to not inline these if it can see that they are called on cold exception throwing paths. I believe that Andy has done some changes recently to make it work even better than before.